sale_commission_pricelist/models/sale_order.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = 'sale.order.line'
+
+    @api.onchange('product_id')
+    def _onchange_sale_commission_pricelist(self):
+        self.ensure_one()
+        if not self.product_id:
+            return
+        rule_id = self.order_id.pricelist_id.get_product_price_rule(
+            product=self.product_id,
+            quantity=self.product_uom_qty or 1.0,
+            partner=self.order_id.partner_id,
+            date=self.order_id.date_order,
+            uom_id=self.product_uom.id)[1]
```

Comment:
Why `[1]`? Does this always return that values?

Reply:
It should.
Return a tuple.

Reply:
But it contains always at least 2 elements?

Reply:
https://github.com/odoo/odoo/blob/f3da958fa144ee1b112df166435a54c064a10812/addons/product/models/product_pricelist.py#L244